sdk/azidentity/azure_cli_credential_client.go

```diff
@@ -0,0 +1,155 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package azidentity
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"os/exec"
+	"regexp"
+	"runtime"
+	"strings"
+	"time"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore"
+)
+
+const (
+	DefaultSuffix        = "/.default"
```

Comment:
Why are these public? Is a customer expected to refer to these strings? If so, why? It feels to me that these would be strings in an error value. Maybe we want public sentinel error values instead? Since there are no comments, I can't discern intent. Please also add comments so that other devs know what you're trying to accomplish here.